src/Utility/AdminNotifications.php

```diff
@@ -160,4 +162,52 @@ public function set_reviewed_status() {
 		}
 		wp_send_json_success( $msg );
 	}
+
+	public function update_available_notice() {
+		if ( ! $this->maybe_show_update_available_notification() ) {
+			return;
+		}
+
+		$url = is_multisite() ? 'network/update-core.php' : 'update-core.php';
+
+		wp_admin_notice(
+			sprintf(
+			/* Translators: placeholders will be html `<a>` links. */
+				esc_html__( 'We wanted to inform you that there is a pending update available for the Constant Contact + WooCommerce plugin. To ensure optimal performance and security, please visit the %1$sWordPress updates%2$s area and update the plugin at your earliest convenience.', 'constant-contact-woocommerce' ),
+				sprintf( '<a href="%s">', esc_url( admin_url( $url ) ) ),
+				'</a>',
+			),
+			[
+				'type'        => 'notice',
+			]
+		);
+	}
+
+	function maybe_show_update_available_notification() { return true;
+		if ( ! current_user_can( 'manage_options' ) ) {
```

Comment:
```suggestion
	function maybe_show_update_available_notification() { 
		if ( ! current_user_can( 'manage_options' ) ) {
```
@tw2113 I'm assuming we don't want to return `true` always, right?

Reply:
Good catch